pycytominer/cyto_utils/util.py

```diff
@@ -124,6 +125,43 @@ def check_consensus_operation(operation):
     return operation
 
 
+def check_fields_of_view_format(fields_of_view):
+    """
+    Confirm that the input fields of view is valid
+
+    Arguments:
+    fields_of_view - "all" or a list of integer fields of view
+
+    Return:
+    Correctly formatted fields_of_view variable
+    """
+
+    if fields_of_view != "all":
+        if isinstance(fields_of_view, list):
+            if all(isinstance(x, int) for x in fields_of_view):
+                return fields_of_view
+            else:
+                raise TypeError(f'Variable of type int expected, however type {type(fields_of_view[0])} was passed')
```

Comment:
i can see this error message frustrating a user if the type of the first element in `fields_of_view` is an int, but one of the other elements is not. Is there a way to account for this in the error message? If not, I don't think we need to try too hard to be specific.

Reply:
Changed it a little - https://github.com/cytomining/pycytominer/pull/129/commits/44f49eb552f9ff9adff012fb01edcd68f89328f7

Reply:
I'm wondering if we should try to first coerce to integers, and then throw an error if it fails. What do you think about that? Maybe a bad idea, IDK

Reply:
Here is my attempt to coerce to integers - https://github.com/cytomining/pycytominer/pull/129/commits/789c6ed3abfe3f4a4fbb8526171448521f5e6401